src/main/resources/assets/admin/common/js/ui/time/Picker.ts

```diff
@@ -203,7 +230,16 @@ export class Picker<T extends Element>
 
         this.initPopup(this.builder);
         this.setupPopupListeners(this.builder);
-        this.initCloseButton();
+        const buttonContainer = new DivEl('btn-container');
+
+        if (this.showDefaultButton) {
+            buttonContainer.appendChild(this.initDefaultButton());
+        }
+
+        const okButton = this.initCloseButton();
```

Comment:
`closeButton` should not be called `okButton`.

Reply:
...and type definition is also required

Reply:
Refactored the ok button to close button. 😃